external/iotivity/iotivity_1.2-rel/extlibs/libcoap/SConscript

```diff
@@ -50,6 +50,15 @@ libcoap_zip_file         = os.path.join(src_dir, 'extlibs', 'libcoap', libcoap_b
 libcoap_url              = libcoap_repo_url + '/archive/' + libcoap_branch + '.zip'
 libcoap_checkout_command = 'git clone ' + libcoap_repo_url + '.git extlibs/libcoap/libcoap --branch ' + libcoap_branch
 
+if GetOption('clean'):
+        lib_path = os.path.join(src_dir, 'extlibs', 'libcoap', 'libcoap.a')
+        print "Library path: %s" %lib_path
+        if os.path.exists(lib_path):
+                result = os.system('rm %s' %lib_path)
+                if result != 0:
+                        print 'Failed to remove libcoap library in extlibs/libcoap'
+        Return('libcoap_env')
```

Comment:
I want to know about Tab Size of Sconscript used in TizenRT. The other part of this file is tab size `space = 4`. However, your patch is `space = 8`.
Another interesting fact is that in other TizenRTs, the tab size is `space = 4` and **TAB** is used. Should iotivity observe the coding rules of TizenRT?

Reply:
Hi @juitem , 
Can you give us a policy on coding style for iotivity ported to TizenRT?

Reply:
There has to be 4 spaces. Thanks for pointing this.
I have updated my patch from `space = 8` to `space = 4`.

Reply:
Even in the build scripts of IoTivity Commercial Repo (https://github.sec.samsung.net/RS7-IOTIVITY/IoTivity), a mixture of TAB and `space = 4` is being used. Currently, It's not uniform.